### PR TITLE
add intel file with filepath and fix tests

### PR DESCRIPTION
**Pull request by pangea-andrest**

### CODE REVIEW

File: packages/pangea-node-sdk/src/services/intel.ts [hunk 1]
```diff
@@ -64,6 +66,40 @@ export class FileIntelService extends BaseService {
...
+   * @example
+   * const options = { provider: "reversinglabs" };
+   * const response = await fileIntel.lookup("142b638c6a60b60c7f9928da4fb85a5a8e1422a9ffdc9ee49e17e56ccca9cf6e", "sha256", options);
+   */
+   lookupFilepath(
+    filepath: string,
+    options: Intel.Options = {}
+  ): Promise<PangeaResponse<Intel.Response>> {
+    const content = readFileSync(filepath);
+    const fileHash = createHash('sha256').update(content).digest('hex')
```
Comment (pangea-nv) [inline packages/pangea-node-sdk/src/services/intel.ts:90]: I would make 'sha256' a const variable at the top of the function or file i.e' 

const hashType = "sha256"; '

then:

    const fileHash = createHash(hashType).update(content).digest('hex')

and:

    const data: Intel.FileParams = {
      hash: fileHash,
      hash_type: hashType,
    };

**Review by pangea-nv — COMMENTED** [review]
looks good. just one minor comment on a best practice

File: packages/pangea-node-sdk/src/services/intel.ts (hunk 1 above)
Comment (pangea-andrest) [reply packages/pangea-node-sdk/src/services/intel.ts]: Updated. At least in this case, that should work. but it could be not strictly the same name

Comment (ggallien13) [reply packages/pangea-node-sdk/src/services/intel.ts]: Makes sense.

**Review by ggallien13 — APPROVED** [review]

**Pull request merged**